### PR TITLE
Update license year

### PR DESCRIPTION
**Pull request by julianacook**
Yay 2017!

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -44,7 +44,7 @@
       of this License, Derivative Works shall not include works that remain
       separable from, or merely link (or bind by name) to the interfaces of,
       the Work and Derivative Works thereof.
- 
+
```
Comment (mbad0la) [inline LICENSE]: Did you insert this newline by mistake?

Comment (gr2m) [inline LICENSE]: @julianacook can you please revert the change above? It looks like there was a space that you removed (or maybe your editor did automatically). GitHub is parsing license files and they are rather particular about its content, so if possible I’d like you to revert that change above and only change the year. You can also close this PR and start another one, if that’s simpler for you? If you can’t help any more no worries, we can finish this up for you, just let us know :)

Comment (julianacook) [inline LICENSE]: @mbad0la It's a space! I've got Atom set up to remove trailing whitespace, so it got rid of this automatically.

@gr2m I don't think the space is necessary for parsing this LICENSE; if you check out some of the other Hoodie LICENSE files, they are identical except don't have this extra space (check out https://github.com/hoodiehq/hoodie-client/blob/master/LICENSE#L47 for example!). Happy to put it back in if y'all prefer, but it shouldn't cause any parsing issues on your side :)

Comment (gr2m) [inline LICENSE]: I fear now you added the space at another place :) I think you are right though, this shouldn’t be any problem. I don’t mind merging as is, but if you could revert the two whitespace changes, that’d be nice

**Comment by mbad0la** [comment]
@gr2m @julianacook merging this in.

Congrats on your first Open Source Contribution! :tada:

**Pull request merged**